staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -376,6 +377,18 @@ var (
 			},
 		},
 	)
+	SuggestedFlowSchemaHealthForStrangers = newFlowSchema(
+		"health-for-strangers", "exempt", 1000,
```

Comment:
AFAIK, downloading the metrics too frequently can also impact the performance of apiserver, make this exempt looks suspicious to me..